widget_recent_window.go

```diff
@@ -50,9 +50,6 @@ func (w *RecentWindowWidget) Update() error {
 	img := image.NewRGBA(image.Rect(0, 0, int(w.dev.Pixels), int(w.dev.Pixels)))
 
 	if int(w.window) < len(recentWindows) {
-		if w.lastID == recentWindows[w.window].ID {
```

Comment:
Why did you remove these lines?

Reply:
Hmm... I must've screwed that up.  Will fix.

Reply:
OK, I remember.  This causes the force update *not* up update the recent windows when the device "wakes up" from sleep

Reply:
Also, `RequiresUpdate` already gates the update on this, so I didn't think this would break things (and it doesn't seem to).

Reply:
Ok, that makes sense!